packages/i18n-calypso/src/index.js

```diff
@@ -6,16 +6,24 @@ import localizeFactory from './localize';
 import translateHookFactory from './use-translate';
 import rtlFactory from './rtl';
 
-const i18n = new I18N();
+// Export the `I18N` class
 export { I18N };
+
+// Create a default instance of `I18N` and make it the module default export
+const i18n = new I18N();
 export default i18n;
+
+// Export the default instance's properties and bound methods for convenience
+// These should be deprecated eventually, exposing only the default `i18n` instance
 export const moment = i18n.moment;
 export const numberFormat = i18n.numberFormat.bind( i18n );
 export const translate = i18n.translate.bind( i18n );
 export const configure = i18n.configure.bind( i18n );
 export const setLocale = i18n.setLocale.bind( i18n );
 export const getLocale = i18n.getLocale.bind( i18n );
 export const getLocaleSlug = i18n.getLocaleSlug.bind( i18n );
+export const getLocaleVariant = i18n.getLocaleVariant.bind( i18n );
+export const isRtl = i18n.isRtl.bind( i18n );
```

Comment:
Bad merge? This change looks unrelated.

Reply:
That's a drive-by change that follows up on #37593 which added the `isRtl` method to the `I18N` class, but I forgot to export the bound version (for consistency's sake)